packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/fragment.js

```diff
@@ -156,49 +155,16 @@ function is_static_element(node) {
 			return false;
 		}
 
-		if (!attribute.metadata.expression.can_inline) {
+		if (
+			attribute.value !== true &&
+			!is_text_attribute(attribute) &&
+			// If the attribute is not a text attribute but is inlinable we will directly inline it in the
+			// the template so before returning false we need to check that the attribute is not inlinable
+			!is_inlinable_expression(attribute.value, state)
```

Comment:
ditto

Reply:
This was there before https://github.com/sveltejs/svelte/pull/14269